src/js/puzzles.js

```diff
@@ -335,6 +335,27 @@ var puzzlesArray = [
   word: 'sun',
   hint: 'Time of my life',
   answer: 'fun'
+},
+{
+  id: 49,
+  sentence: 'The star is out of my peach.',
+  word: 'peach',
+  hint: 'beyond the limit',
```

Comment:
This hint talks about the sentence itself and not the correct word. Can you find a better way to describe "reach"? "Within arms-length" or something similar?

Reply:
Updated as requested